code/game/gamemodes/gang/dominator.dm

```diff
@@ -19,10 +20,12 @@
 	..()
 	SetLuminosity(2)
 	poi_list |= src
+	spark_system = new
+	spark_system.set_up(5, 1, src)
 
 /obj/machinery/dominator/examine(mob/user)
 	..()
-	if(operating == -1)
+	if(stat & BROKEN)
```

Comment:
we have a BROKEN bit flag in stat, so let's use it instead of making the operating var weird.